mne/viz/topo.py

```diff
@@ -437,16 +446,16 @@ def _plot_timeseries_unified(bn, ch_idx, tmin, tmax, vmin, vmax, ylim, data,
     """Show multiple time series on topo using a single axes."""
     import matplotlib.pyplot as plt
     if not (ylim and not any(v is None for v in ylim)):
-        ylim = np.array([np.min(data), np.max(data)])
+        ylim = [min(np.min(d) for d in data), max(np.max(d) for d in data)]
```

Comment:
why do you need this? np.min(data) should do the job

Reply:
`data` can be different sizes
```
>>> np.min([[1, 2], [3]])
[1, 2]
```
So you have to iterate over the `data` entries, which return singletons, then min over those

Reply:
ok ! now we just need a test then :)

Reply:
I was afraid that @agramfort would ask for a test ;)
Will be coming soon.